Drv/LinuxSpiDriver/LinuxSpiDriverComponentImpl.cpp

```diff
@@ -41,14 +41,14 @@ namespace Drv {
             return;
         }
 
-        DEBUG_PRINT("Writing %d bytes to SPI\n",writeBuffer.getsize());
+        DEBUG_PRINT("Writing %d bytes to SPI\n",writeBuffer.getSize());
 
         spi_ioc_transfer tr;
         // Zero for unused fields:
         memset(&tr, 0, sizeof(tr));
-        tr.tx_buf = writeBuffer.getdata();
-        tr.rx_buf = readBuffer.getdata();
-        tr.len = writeBuffer.getsize();
+        tr.tx_buf = (U64)writeBuffer.getData();
```

Comment:
Should we update the SPI driver to U8*?

Reply:
This type is defined as U64 in the Linux SPI header.  It is not in our control?